src/test/java/MapTest.java

```diff
@@ -0,0 +1,13 @@
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MapTest {
+    @Test
+    public void checkMapTest() {
+        MapExample.characterCount("test");
```

Comment:
Please remove this line or extract it to some variable and then use it in your assertion

Reply:
Line was delited